### PR TITLE
travis ci corrections

### PR DESCRIPTION
**Pull request by duttonw**

### CODE REVIEW

File: bin/travis-build.bash [hunk 1]
```diff
@@ -1,75 +1,123 @@
...
+    CKAN_TAG=$(git ls-remote --tags https://github.com/$CKAN_GIT_REPO | grep refs/tags/ckan-$CKANVERSION | awk '{print $2}'| sort --version-sort | tail -n 1 | sed  's|refs/tags/||' )
+    echo "CKAN tag version $CKANVERSION is: ${CKAN_TAG#ckan-}"
+    cmd="git clone --depth=50 --branch=$CKAN_TAG https://github.com/$CKAN_GIT_REPO ckan"
+    echo $cmd
+    eval $cmd
+  else
+    echo "CKAN version: $CKAN_BRANCH"
+    cmd="git clone --depth=50 --branch=$CKAN_BRANCH https://github.com/$CKAN_GIT_REPO ckan"
+    echo $cmd
+    eval $cmd
```
Comment (ThrawnCA) [inline bin/travis-build.bash]: If we're using `eval`, then should we quote `$cmd`?

**Review by ThrawnCA — APPROVED** [review]
@duttonw I'm not certain of how to apply the depth-50 changes to ckanext-datarequests, which doesn't include any calls to "git clone".

File: bin/travis-build.bash (hunk 1 above)
Comment (duttonw) [reply bin/travis-build.bash]: should drop this down again, as its not required post testing

File: bin/travis-build.bash [hunk 2]
```diff
@@ -41,6 +41,10 @@ fi
 
 pushd ckan
 
+if [ $ver -eq 3 ]; then
+pip install -r requirement-setuptools.txt
```
Comment (ThrawnCA) [inline bin/travis-build.bash:41]: Should be indented.

**Review by ThrawnCA — APPROVED** [review]

**Pull request merged**